os/arch/arm/src/armv7-a/arm_cpustart.c

```diff
@@ -111,6 +112,9 @@ int arm_start_handler(int irq, void *context, void *arg)
 
 	svdbg("CPU%d Started\n", this_cpu());
 
+	/* start generic timer PPI for this core */
+	up_timer_initialize();
```

Comment:
It is called in up_initialize(). Please check it is not problem

Reply:
Please check whether it is ok although it is called multiple times or add conditions to avoid duplicated calls.